.eslintrc

```diff
@@ -16,6 +16,7 @@
     "no-use-before-define": 2,
     "radix": 2,
     "semi": 2,
+    "block-spacing": 2,
```

Comment:
Added this to make sure there is spacing around blocks in the future. i.e.

```js
if (err) { throw err; }
```

instead of

```js
if (err) {throw err;}
```